### PR TITLE
build: add RxJava dependency

### PR DESCRIPTION
**Pull request by walkingice**
To involve RxJava to code base

### CODE REVIEW

**Comment by walkingice** [comment]
Per my build test

the release apk size is
**2371662** - before adding dependency
**2402586** - after adding dependency includes below changes

```java
Observable.just("one", "two", "three", "four", "five")
         .subscribeOn(Schedulers.newThread())
         .observeOn(AndroidSchedulers.mainThread())
          .subscribe(new Consumer<String>() {
             @Override
             public void accept(String s) throws Exception {
                 Log.d("TAG", s);
              }
         });
```

**Comment by idiotmax** [comment]
How about let's make MainActivity.asyncInitialize as a real use case?

File: app/build.gradle [hunk 1]
```diff
@@ -213,6 +213,10 @@ dependencies {
     implementation Libs.adjust
     implementation Libs.play_services_analytics // Required by Adjust
 
+    // https://github.com/ReactiveX/RxAndroid
+    implementation 'io.reactivex.rxjava2:rxandroid:2.0.2'
```
Comment (benjamin-cheng) [inline app/build.gradle]: Could you please help to define the dependency in the file below?
https://github.com/mozilla-tw/Rocket/blob/master/buildSrc/src/main/java/Dependencies.kt

Then you could reference here as:
`implementation Libs.rxandroid`
`implementation Libs.rxjava`

**Comment by walkingice** [comment]
We had kotlin, and we don't have plan for RxJava for now.

**Pull request closed without merging**